app/assets/javascripts/home.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() {
+
+  if ($('#add').text() == "Add Menu") {
+    $('#home-form').hide();
+  };
+
+  $('.page-content').click(function(){
+    $('#add').show();
+    $('#home-form').hide();
+    if ($('#error').text() == "Menu name can't be blank") {
+      $('#add').show();
+      $('#error').hide();
+      $(".footer ul").css({marginTop: 0});
+    };
+  });
+
+  if ($('#error').text() == "Menu name can't be blank") {
+    $('#add').last().hide();
+    $('#home-form').show();
+    $(".footer .form").css({padding: 0});
+  };
+
+  $('#add').last().click(function(){
+    $(this).hide();
+    $(".footer .form").css({padding: 20});
+    $('#home-form').show();
```

Comment:
Regarding the add/removing of padding and margin when showing/hiding different parts of this. I would try to get away from using javascript to do this. It will become difficult to maintain and debug. I would recommend instead adding styling with classes, then just add and remove classes depending on the state it's in.